sql/o.karmanskyi_homework_26.sql

```diff
@@ -0,0 +1,46 @@
+CREATE DATABASE HomeworkDB;
+
+CREATE TABLE HomeworkDB.dbo.PhoneBook
+(
+	Id INT IDENTITY(1, 1) NOT NULL PRIMARY KEY,
+	UserFirstName VARCHAR(20) NOT NULL,
+	UserLastName VARCHAR(20),
```

Comment:
I think LastName should be required as well (how to identify person only by name :) )